src/ButtonBack/__tests__/ButtonBack.test.jsx

```diff
@@ -207,4 +207,20 @@ describe('<ButtonBack />', () => {
     wrapper.find('button').simulate('click');
     expect(wrapper.instance().getStore().state.isPlaying).toBe(false);
   });
+  it('should use given tag instead of button', () => {
```

Comment:
I think we are leaving out a test for use of intrinsic elements?

Reply:
Agreed, I will implement a test for all cases

Reply:
A test for intrinsic tag was done for all affected components